### PR TITLE
Add newline config to vscode settings

### PR DESCRIPTION
**Pull request by rajraj**
Adding these to the vscode settings,

```
"files.insertFinalNewline": true,
"editor.scrollBeyondLastLine": false
```

### CODE REVIEW

**Review by csfh — APPROVED** [review]

File: configs/vscode.json [hunk 1]
```diff
@@ -22,5 +22,7 @@
   "explorer.confirmDelete": false,
   "extensions.ignoreRecommendations": true,
   "workbench.colorTheme": "Tokyo Night",
-  "window.titleBarStyle": "custom"
+  "window.titleBarStyle": "custom",
+  "files.insertFinalNewline": true,
+  "editor.scrollBeyondLastLine": false
```
Comment (florentdestremau) [inline configs/vscode.json:27]: while I'm all for insertFinalNewLine, scrollBeyongLastLine is not necessary IMO

Comment (csfh) [reply configs/vscode.json:27]: Fair enough 👍🏻

**Comment by dhh** [comment]
Hmm, I think this is too specific. We generally don't use a trailing new line.

**Comment by rmacklin** [comment]
@dhh @jeremy I'm a little confused re: "we generally don't use a trailing new line" - isn't ensuring a trailing newline the default behavior when writing a file in vim & neovim? (So my interpretation of this PR was to make vscode consistent in following that unix convention. That way if you create a file in vscode, which is later edited in neovim, there wouldn't suddenly be an unrelated diff adding the trailing newline.)

**Pull request closed without merging**